### PR TITLE
Adds config note warning that TGUI CDNs do not work unless CORS headers are provided

### PR DESCRIPTION
**Pull request by ghost**
Adds config note warning that TGUI CDNs do not work unless CORS headers are provided

## About The Pull Request

Adds a note warning that CORS headers are required for TGUI CDNs

## Why It's Good For The Game

I spent many hours trying to figure this out, including asking around, and no one seemed to know how to fix it. I think it would be good to note this for anyone else that tries to set up a TGUI CDN.

## Changelog

:cl:
server: Updated instructions on setting up a TGUI CDN, noting that TGUI via CDN malfunctions in the event CORS headers are absent.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
reframe your changelog to use the `server:` tag, and make it more pertinent to information a server operator might want to know, e.g. "Updated instructions on setting up a TGUI CDN to be more sufficient." or something along those lines? Not really useful to frame it as a config thing since a CDN is much more server-ops aligned than config stuff

could be speaking out of my ass but i think changing the changelog notes to reflect that isn't the worst idea in the world. Thanks for passing along the information though!

**Comment by YehnBeep** [comment]
Updated

**Comment by optimumtact** [comment]
thank you, this is something that lives in my head rent free but I never wrote down

File: config/resources.txt [hunk 1]
```diff
@@ -39,3 +39,8 @@ EXTERNAL_RSC_URLS http://tgstation13.download/byond/tgstationv2.zip
 # if you want to test this locally, you simpily run the `localhost-asset-webroot-server.py` python3 script to host assets stored in `data/asset-store/` via http://localhost:58715/
 #ASSET_CDN_URL http://localhost:58715/
 
+# CDN webservers must provide CORS headers, or TGUI will break and not be able to load all elements. The following example is for IIS:
+# <add name="Access-Control-Allow-Origin" value="*" />
+# <add name="Vary" value="Origin" />
+# <add name="Access-Control-Allow-Methods" value="GET, POST, OPTIONS" />
```
Comment (optimumtact) [inline config/resources.txt:45]: you probably don't need to allow post right? there is no outbound data being transferred

Comment (YehnBeep) [reply config/resources.txt:45]: I don't know. If you know for certain, that could be a refinement added.

My level of knowledge is being stuck on this for a long time with a webserver that was definitely sending files but that TGUI would only use some of - and eventually finding out that adding the above header made it work. I don't know for certain/exactly what portions are needed or might not be.

Comment (optimumtact) [reply config/resources.txt:45]: yeah no worries

**Pull request merged**